server/src/main/scala/za/co/absa/atum/server/api/repository/PartitioningRepositoryImpl.scala

```diff
@@ -61,25 +55,35 @@ class PartitioningRepositoryImpl(
     IO[DatabaseError, AdditionalDataDTO] = {
     getPartitioningAdditionalDataFn(partitioning).mapBoth(err => DatabaseError(err.getMessage), _.toMap)
   }
+
+  override def getPartitioningCheckpoints(partitioningName: CheckpointQueryDTO):
+  IO[DatabaseError, Seq[CheckpointMeasurements]] = {
+    dbCall(getPartitioningCheckpointsFn(partitioningName).mapError(
```

Comment:
is this `mapError` necessary? `.mapError(error => DatabaseError(error.getMessage))` is already in `dbCall`

Reply:
No, it is not if it is in the `dbCall`.